github/ci/prow-deploy/kustom/base/manifests/test_infra/1736ab42da959a5be7de22586b495120000bc4ef/needs-rebase_deployment.yaml

```diff
@@ -32,7 +32,7 @@ spec:
       terminationGracePeriodSeconds: 180
       containers:
       - name: needs-rebase
-        image: gcr.io/k8s-prow/needs-rebase:v20201020-a5b49bc903
+        image: gcr.io/k8s-prow/needs-rebase:v20210217-8c724915ff
```

Comment:
Just being curious: is there a reason why the [image from here](https://github.com/kubevirt/project-infra/pull/960/files#diff-3ed8f33769ef8ad2f23415112f7af080043c5ddee8510e7c11a8a6b37b069b37R128) is a week older (the images near this one all have the same tag)

Reply:
Good catch! No idea sorry, this comes verbatim from upstream manifests at `1736ab42da959a5be7de22586b495120000bc4ef` https://github.com/kubernetes/test-infra/blob/1736ab42da959a5be7de22586b495120000bc4ef/config/prow/cluster/needs-rebase_deployment.yaml#L35